### PR TITLE
Arm32 support

### PR DESCRIPTION
**Pull request by etienne-lms**
Supersedes https://github.com/ARM-software/bbr-acs/pull/8.

### CODE REVIEW

File: common/scripts/build-fwts-arm.sh [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    echo $BBR_DIR
+
+    if [ "$BUILD_PLAT" = "IR" ]; then
+        if ! patch -R -s -f --dry-run -p1 < $BBR_DIR/ebbr/config/IR_VER.patch ; then
+        echo "Applying FWTS Patch ..."
+        patch -p1 < $BBR_DIR/ebbr/config/IR_VER.patch
+        fi
+    fi
+    if [ "$BUILD_PLAT" = "ES" ]; then
```
Comment (vstehle) [inline common/scripts/build-fwts-arm.sh:87]: The SystemReady ES band accepts only 64b systems for now so I think you can drop support for ES+32b.

File: common/scripts/build-sct-arm.sh [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+. $TOP_DIR/../../common/scripts/cross_toolchain-arm.sh
+
+BUILD_PLAT=$1
+BUILD_TYPE=$2
+
+if [ $BUILD_PLAT = SR ]; then
+   BUILD_PLAT=ES
+fi
+
+if ! [[ $BUILD_PLAT = IR ]] && ! [[ $BUILD_PLAT = ES ]] ; then
```
Comment (vstehle) [inline common/scripts/build-sct-arm.sh:67]: Same as previous comment: you can drop references to ES and SR for 32b.

File: common/scripts/build-sct-arm.sh [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+    mkdir -p ${TARGET_ARCH}_SCT/SCT
+    cp -r Build/bbrSct/${UEFI_BUILD_MODE}_${UEFI_TOOLCHAIN}/SctPackage${TARGET_ARCH}/${TARGET_ARCH}/* ${TARGET_ARCH}_SCT/SCT/
+
+    if [ $BUILD_PLAT = IR ]; then
+        #EBBR
+        cp Build/bbrSct/${UEFI_BUILD_MODE}_${UEFI_TOOLCHAIN}/SctPackage${TARGET_ARCH}/EBBRStartup.nsh ${TARGET_ARCH}_SCT/SctStartup.nsh
+        cp SctPkg/BBR/EfiCompliant_EBBR.ini ${TARGET_ARCH}_SCT/SCT/Dependency/EfiCompliantBBTest/EfiCompliant.ini
+        cp SctPkg/BBR/EBBR_manual.seq ${TARGET_ARCH}_SCT/SCT/Sequence/EBBR_manual.seq
+    else
```
Comment (vstehle) [inline common/scripts/build-sct-arm.sh:170]: Ditto: only IR does support 32b, all other cases can be dropped.

File: common/scripts/build-uefi-apps-arm.sh [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+BUILD_PLAT=$1
+BUILD_TYPE=$2
+
+if [ $BUILD_PLAT = SR ]; then
+   BUILD_PLAT=ES
+fi
+
+#Currently the BUILD_PLAT flag is not used. For future use
+if ! [[ $BUILD_PLAT = IR ]] && ! [[ $BUILD_PLAT = ES ]] ; then
```
Comment (vstehle) [inline common/scripts/build-uefi-apps-arm.sh:67]: Ditto: you can drop all non-IR for 32b.

File: common/scripts/build-fwts-arm.sh (hunk 1 above)
Comment (etienne-lms) [reply common/scripts/build-fwts-arm.sh:87]: Ok, i'll remove instruction related to ES.

Comment (etienne-lms) [reply common/scripts/build-fwts-arm.sh]: addressed

Comment (etienne-lms) [reply common/scripts/build-fwts-arm.sh]: addressed in [`[review]` fixup commit](https://github.com/ARM-software/bbr-acs/pull/11/commits/1e150ed066f2c2bb0693721d0cada03c71abd8cb#diff-cb7f13fe0d41d1fef7525b4f7ae1c8a671d25d9df797093c76005cfdf1d22a0cR51).

**Pull request merged**